webapp/app/templates/content/data_privacy.html

```diff
@@ -54,31 +54,31 @@ <h2 class="pb-1">{{ _('data_privacy.rights-title') }}</h2>
         <p>{{ _('data_privacy.rights-text') }}</p>
 
         <div class="mt-5">
-            <h4>{{ _('data_privacy.rights-information') }}</h4>
+            <h3 class="h4">{{ _('data_privacy.rights-information') }}</h3>
```

Comment:
Although the use of the h4 class is clever, I am not such a big fand to be honest. I would say that we probably should control the look through our own class and use our variables there. We might not have to do this now but it seems cleaner to me.

WDYT?

Reply:
I mean, we can just override the bootstrap classes if we want them to look another way?

Reply:
We talked about it and I understood the implications of it. However, we decided to leave it at that for now and get to it later at a larger scale when splitting up the css files / using sass instead. I added a comment to the corresponding ticket.